spec/models/reward_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Reward, type: :model do
+  subject(:reward) { described_class.new(title: 'test', description: 'test', price: 1) }
```

Comment:
I would suggest using FactoryBot with `build`

```suggestion
  let(:reward) { build(:reward, title: 'test', description: 'test', price: 1) }
```